slybot/slybot/starturls/feed_generator.py

```diff
@@ -0,0 +1,15 @@
+from scrapy import Request
```

Comment:
Can the slybot changes be moved to a different PR so they can be released on PyPi